pkg/sdkclient/grpc_resolver/client_resolver.go

```diff
@@ -80,14 +82,17 @@ func (*multiProcResolver) Close()                                  {}
 func (*multiProcResolver) Resolve(target resolver.Target)          {}
 
 // buildConnAddrs Populate the connection list for the clients
-// Format (serverAddr, serverIdx) : (0.0.0.0:5551, 1), (0.0.0.0:5552, 2)
-func buildConnAddrs(servPorts []string) []string {
-	var conn = make([]string, len(servPorts))
-	for i := 0; i < len(servPorts); i++ {
-		// Use the server ports from the server info file and assign to each client
-		addr, _ := strconv.Atoi(servPorts[i])
+// Format (serverAddr, serverIdx) :
+// (unix:///var/run/numaflow/multiproc0.sock, 1),
```

Comment:
Why is 0 mapping to idx 1 but 2 is mapped to idx 2? doc typo?

Reply:
Typo. Fixed, thanks for pointing out

Reply:
What was the reason we originally use 0 -> 1, 1 -> 2, instead of 0 -> 0, 1 -> 1?

Reply:
That was from legacy code. Have updated to match server name with indexes